Dockerfile

```diff
@@ -21,6 +21,9 @@ RUN \
 
 # CLI utilities
 RUN yum install -y gcc gcc-c++ git make openssl unzip wget zip jq
+RUN git clone https://github.com/nasa/cumulus-dashboard.git && chmod 777 /cumulus-dashboard
```

Comment:
I'd use 755 instead of 777 it's just probably not needed to give rwx for everyone just bad security hygiene

Reply:
I did not mean to include that... Good catch.

Reply:
I am a bit confused how that got in here. That was me trying to figure out why I couldn't get the Dashboard working the other day.